tinker/templates/e-announcements/results.html

```diff
@@ -15,13 +15,22 @@
             </p>
             <div class="large-6 columns small-3">
                 <ul class="btn-group-horizontal">
+                {% set first_date = form.first_date.split(" ") -%}
+
+                <p>{{ first_date }} {{ hour_today }} {{ tomorrow }}</p>
                     {% if form.workflow_status %}
                         <li>This announcement is pending approval.</li>
                     {% else %}
                         {% if not form.first_date_past and not form.second_date_past %}
                             <li>
-                                <a href="{{ url_for('EAnnouncementsView:edit', e_announcement_id=form.id) }}"
-                                   class="btn btn-primary">Edit</a></li>
+                                {% if (first_date[1] != tomorrow.month) or (first_date[1] == tomorrow.month and first_date[2] != tomorrow.day) %}
```

Comment:
This logic might be able to be cleaned up further. If its past 1pm of either of the dates, then the entire e-annz is uneditable, I believe. Then if its between 1pm and the final date, maybe we display a message letting them know they can't edit? And if its after we do what we normally do (i don't remember off the top of my head).

Reply:
I simplified the code and added a message if after 1pm the day before an e-annz is sent.